src/test/java/edu/ucsb/mapache/documents/ChannelTopicTests.java

```diff
@@ -0,0 +1,64 @@
+package edu.ucsb.mapache.documents;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Assertions;
+
+import org.junit.jupiter.api.Test;
+
+public class ChannelTopicTests {
+
+  @Test
+  public void test_toString() {
+    ChannelTopic ct = new ChannelTopic("sampleTopic", "sampleCreator");
+    String expected = "{ value='sampleTopic', creator='sampleCreator'}";
+    assertEquals(expected, ct.toString());
+  }
+
+  @Test
+  public void test_getAndSetValue() {
+    ChannelTopic ct = new ChannelTopic();
+    ct.setValue("sampleTopic");
+    assertEquals("sampleTopic", ct.getValue());  
+  }
+
+  @Test
+  public void test_getAndSetCreator() {
+    ChannelTopic ct = new ChannelTopic();
+    ct.setCreator("sampleCreator");
+    assertEquals("sampleCreator", ct.getCreator());  
+  }
```

Comment:
We should use `JavaBeanTester` here to reduce boilerplate tests to a single test.

Reply:
Done